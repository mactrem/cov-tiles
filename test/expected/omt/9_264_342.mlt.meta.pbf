Ã

water_name
id" 
geometry* 
class" 	
intermittent" 
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
name:ar" 	
name:uk" 	F
	landcover
id" 
geometry* 
class" 	
subclass" 	ž
waterway
id" 
geometry* 
class" 	
name" 	
name:de" 	
name:en" 	
name:fy" 	
name:nl" 	
name:fr" 	
name:ru" 	
name:es" 	
name:hu" 	
name:it" 	
name:ja" 	
name:pl" 	
name:sk" 	
name:uk" 	2
landuse
id" 
geometry* 
class" 	¹
mountain_peak
id" 
geometry* 
class" 	
ele" 
ele:ft" 
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
rank" ´
place
id" 
geometry* 
class" 	
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
rank" 
capital" 
name:ar" 	
name:eo" 	
name:es" 	
name:fr" 	
name:fy" 	
name:he" 	
name:la" 	
name:lt" 	
name:mk" 	
name:nl" 	
name:ru" 	
name:sr" 	
name:uk" 	
name:zh" 	
name:be" 	
name:el" 	
name:ja" 	
name:bg" 	
name:hu" 	
name:ka" 	
name:kk" 	
name:ko" 	
name:lv" 	
name:th" 	
name:id" 	
name:ca" 	
name:pt" 	
name:az" 	
name:hy" 	
name:eu" 	
name:it" 	F
water
id" 
geometry* 
class" 	
intermittent" Æ
transportation_name
id" 
geometry* 
class" 	
network" 	
ref" 	

ref:length" 
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	–
park
id" 
geometry* 
class" 	
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
name:nl" 	§
transportation
id" 
geometry* 
class" 	
layer" 
subclass" 	
brunnel" 	
bicycle" 	
foot" 	
horse" 	
ramp" 