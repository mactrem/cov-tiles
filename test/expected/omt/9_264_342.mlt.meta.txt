version: 1
featureTables {
  name: "water_name"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "intermittent"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ar"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:uk"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "landcover"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "waterway"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:fy"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:nl"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:fr"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ru"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:es"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:hu"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:it"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ja"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:pl"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:sk"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:uk"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "landuse"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "mountain_peak"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_64
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ele"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "ele:ft"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "rank"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "place"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_64
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "rank"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "capital"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name:ar"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:eo"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:es"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:fr"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:fy"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:he"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:la"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:lt"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:mk"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:nl"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ru"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:sr"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:uk"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:zh"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:be"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:el"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ja"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:bg"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:hu"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ka"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:kk"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ko"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:lv"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:th"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:id"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ca"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:pt"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:az"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:hy"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:eu"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:it"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "water"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "intermittent"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "transportation_name"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "network"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ref"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ref:length"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "park"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:nl"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "transportation"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "layer"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "brunnel"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "bicycle"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "foot"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "horse"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ramp"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
